From f9ba7de1f6478d6d8f93c14b3801bfeceb62223b Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Fri, 1 Dec 2023 16:21:03 +0800
Subject: [PATCH 12/14] tensorflow: lock version conflict [1/1]

PD#SWPL-148061

Problem:
Fetch tensorflow v2.6.0 conflict while 202311 release

Solution:
lock version as commit

Fixed conflicts:
NA

Verify:
local

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: I441ef5f157e84e27f7b37e884fefc96d8a6b11b2
---
 recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.6.0.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.6.0.bb b/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.6.0.bb
index 0844888c..4c4d9fe6 100644
--- a/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.6.0.bb
+++ b/recipes-tensorflow/tensorflow-lite/tensorflow-lite_2.6.0.bb
@@ -24,7 +24,7 @@ SHA256SUM_PSIMD = "dc615342bcbe51ca885323e51b68b90ed9bb9fa7df0f4419dbfa0297d5e83
 
 DEPENDS = "zlib"
 SRC_URI = " \
-    git://github.com/tensorflow/tensorflow;destsuffix=git/;branch=r2.6;rev=v2.6.0;protocol=https \
+    git://github.com/tensorflow/tensorflow;destsuffix=git/;branch=r2.6;rev=919f693420e35d00c8d0a42100837ae3718f7927;protocol=https \
     https://gitlab.com/libeigen/eigen/-/archive/7b35638ddb99a0298c5d3450de506a8e8e0203d3/eigen-7b35638ddb99a0298c5d3450de506a8e8e0203d3.tar.gz;sha256sum=${SHA256SUM_EIGEN} \
     https://github.com/abseil/abseil-cpp/archive/997aaf3a28308eba1b9156aa35ab7bca9688e9f6.tar.gz;sha256sum=${SHA256SUM_ABSL} \
     https://github.com/google/gemmlowp/archive/fda83bdc38b118cc6b56753bd540caa49e570745.zip;sha256sum=${SHA256SUM_GEMMLOWP} \
-- 
2.25.1

